_posts/fr/2022-03-22-enquete-sur-le-crypto-gaming.md

```diff
@@ -77,7 +77,7 @@ L’éditeur va ensuite proposer aux utilisateurs de gagner des objets sous form
 Dans beaucoup de cas, plus le jeu offrira des avantages à jouer, plus le jeu sera attractif, plus il y aura de joueurs. Et les joueurs ayant investis rendront le jeu attractif pour ceux qui veulent se développer ou les nouveaux : ce sont eux qui achètent leurs jetons, et qui vont leur permettre de gagner de l’argent.
 
 <div style="text-align: center;">
-    <img src="{{ site.baseurl }}/assets/2022-03-16-enquete-sur-le-crypto-gaming/business.png" width="700px" alt="Starting game project" style="display: block; margin: auto;"/>
+    ![]({{ site.baseurl }}/assets/2022-03-16-enquete-sur-le-crypto-gaming/business.png) width="700px" alt="Starting game project" style="display: block; margin: auto;"/>
```

Comment:
```suggestion
    ![]({{ site.baseurl }}/assets/2022-03-16-enquete-sur-le-crypto-gaming/business.png?width="700")
```

Reply:
I did not put the width of 700 because the difference is hardly visible visually